packages/account/src/Components/forms/idv-form.tsx

```diff
@@ -0,0 +1,303 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Field, FormikValues, FormikProps, FormikHandlers } from 'formik';
+import { ResidenceList } from '@deriv/api-types';
+import { localize } from '@deriv/translations';
+import { formatInput, IDV_NOT_APPLICABLE_OPTION } from '@deriv/shared';
+import { Autocomplete, DesktopWrapper, Input, MobileWrapper, SelectNative, Text } from '@deriv/components';
+import { getDocumentData, preventEmptyClipboardPaste, generatePlaceholderText } from 'Helpers/utils';
+
+type TDocumentList = Array<{
+    id: string;
+    text: string;
+    value?: string;
+    sample_image?: string;
+    example_format?: string;
+    additional?: any;
+}>;
+
+type TFormProps = {
+    document_type: TDocumentList[0];
+    document_number: string;
+    document_additional: string;
+};
+
+type TIDVForm = {
+    selected_country: ResidenceList[0];
+    hide_hint?: boolean;
+    class_name?: string;
+} & FormikHandlers &
+    FormikProps<TFormProps>;
+
+const IDVForm = ({
+    errors,
+    touched,
+    values,
+    handleBlur,
+    handleChange,
+    setFieldValue,
+    class_name,
+    ...props
```

Comment:
Can you list out all the props instead of doing this ?

cc: @yashim-deriv

Reply:
Will make the change